docker-java/template.mf

```diff
@@ -13,7 +13,7 @@ Import-Template:
  javax.ws.rs.*;version="[2.0.0, 2.1.0)",
  org.apache.commons.compress.*;version="${commons-compress.version:short}",
  org.apache.commons.io.*;version="${commons-io.version:short}",
- org.apache.commons.lang.*;version="${commons-lang.version:short}",
+ org.apache.commons.lang33.*;version="${commons-lang3.version:short}",
```

Comment:
Shouldn't this be?
```suggestion
 org.apache.commons.lang3.*;version="${commons-lang3.version:short}",
```

Reply:
Yes, of course, unfortunate search/replace